package.json

```diff
@@ -58,6 +60,7 @@
     "jsonwebtoken": "8.5.1",
     "lexical": "^0.5.0",
     "lint-staged": "13.0.3",
+    "lodash": "4.17.21",
```

Comment:
lodash is a pretty large dependency. We should follow the existing example in the package.json and only install what we need https://www.npmjs.com/package/lodash.get

Reply:
In the code where I used the `get` I am only importing that specific function from Lodash. Is not it just loading what is required thus not the whole library?

Reply:
Or maybe you are just worried about the size of our client package at the end?

Reply:
Yes worried about the size of the client package at the end . Im not sure if webpack would bundle lodash in its entirety

Reply:
removed lodash and replaced with lodash.get